README.md

```diff
@@ -1,70 +1,169 @@
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Learn More
+<!-- LIVE DEMO -->
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## 🚀 Live Demo <a name="live-demo"></a>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+> Add a link to your deployed project.
 
-### Code Splitting
+- [Live Demo Link]()
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Analyzing the Bundle Size
+<!-- GETTING STARTED -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Making a Progressive Web App
+> Describe how a new developer could make use of your project.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+To get a local copy up and running, follow these steps.
 
-### Advanced Configuration
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+In order to run this project you need:
+- Node js
+- A browser
+- Any code editor (vs code)
+
+### Setup
+
+Clone this repository to your desired folder:
+- `https://github.com/collins-kiprotich/maths-magician.git`
+
+
+### Install
+
+Install this project with:
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+- for build `npm run build`
+- for development `npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+- `npm run test`
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Collins Kiprotich**
+
+- GitHub: [Collins-kiprotich](https://github.com/collins-kiprotich)
+- Twitter: [@jcee](https://twitter.com/Jcee01188383)
+- LinkedIn: [Collins Kiprotich](https://www.linkedin.com/in/collins-kiprotich-152b52246/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] [optional] @collins-kiprotich It would be nice if you added future features in your readme file